region/lowernorfair/east/Lower Norfair Spring Ball Maze Room.json

```diff
@@ -727,6 +757,11 @@
       "clearsObstacles": ["B"],
       "note": "Jump and Morph with a speedball to enter the morph tunnel and then use SpringBall to break the bomb blocks."
     },
+    {
+      "link": [6, 6],
```

Comment:
Yeah, and as we continue to refine the model, making it more accurate, we can always loosen the heat damage multipliers to compensate for a loss of "accidental" leniency coming from model imperfections. Relying on model imperfections for leniency is unreliable, since some strats have more imperfection than others. More accuracy should allow us to control the difficulty level more consistently.